cmd/gobl/build_test.go

```diff
@@ -181,6 +181,22 @@ func Test_build(t *testing.T) {
 				draft: false,
 			},
 		},
+		{
+			name: "set to draft on non-envelop",
+			args: []string{"testdata/envelop.nototals.json"},
```

Comment:
`envelop.nototals.json` is a bit confusing, but this testfile actually **doesn't** contain an envelop.